tools/wptserve/wptserve/response.py

```diff
@@ -426,7 +425,7 @@ def end_headers(self):
 
     def write_content(self, data):
         """Write the body of the response."""
-        if isinstance(data, types.StringTypes):
+        if isinstance(data, (text_type, binary_type)):
```

Comment:
Can you add a comment saying these are deliberately both allowed and to see `self.encode`, given this is a bit unusual?

Reply:
Done, below this line.